dotfiles/crictl.yaml

```diff
@@ -0,0 +1,4 @@
+runtime-endpoint: unix:///run/containerd/containerd.sock
+image-endpoint: unix:///run/containerd/containerd.sock
+timeout: 10
+debug: true
```

Comment:
Do you know what this configuration parameter is used for?

Reply:
i got this sample from https://github.com/kubernetes-sigs/cri-tools/blob/master/docs/crictl.md - i haven't used crictl before so do let me know if i need to remove some of the configs...

Reply:
Apparently the flag --debug enables debug output for crictl itself. So lets not use include that per default.

Reply:
thanks, i've removed debug line from the config